scico/plot.py

```diff
@@ -330,7 +330,7 @@ def surf(
             z-axis. An int specifies the number of contours to plot, and
             a sequence specifies the specific contour levels to plot.
         cmap (:class:`matplotlib.colors.Colormap` object, optional (default None)):
-            Colour map for surface. If none specifed, defaults to ``cm.YlOrRd``.
+            Colour map for surface. If none specifed, defaults to `cm.YlOrRd`.
```

Comment:
Shouldn't we use the American spelling, `color`?

Reply:
Spelling changed.